### PR TITLE
Change Google Developers Console Section

### PR DESCRIPTION
**Pull request by nkhalili**
Since Google Developers Console Interface has been changed we need to update this section.

### CODE REVIEW

**Comment by Rick-Anderson** [comment]
@ardalis  please review for PO work

**Comment by Rick-Anderson** [comment]
@nkhalili  please check my changes in the 2nd commit.

File: aspnet/mvc/overview/security/create-an-aspnet-mvc-5-app-with-facebook-and-google-oauth2-and-openid-sign-on.md [hunk 1]
```diff
@@ -110,16 +110,16 @@ To connect to authentication providers like Google and Facebook, you will need t
 ## Creating a Google app for OAuth 2 and connecting the app to the project
 
 1. Navigate to the [Google Developers Console](https://console.developers.google.com/).
-2. Click the **Create Project** button and enter a project name and ID (you can use the default values). In a few seconds the new project will be created and your browser will display the new projects page.
-3. In the left tab, click **APIs &amp; auth**, and then &gt; **Credentials**.
-4. Click the **Create New Client ID** under OAuth. 
+1. If you haven't create a project before, select **Credentials** in the left tab, and then select **Create**.
```
Comment (ardalis) [inline aspnet/mvc/overview/security/create-an-aspnet-mvc-5-app-with-facebook-and-google-oauth2-and-openid-sign-on.md]: create->created

File: aspnet/mvc/overview/security/create-an-aspnet-mvc-5-app-with-facebook-and-google-oauth2-and-openid-sign-on.md [hunk 2]
```diff
@@ -110,16 +110,16 @@ To connect to authentication providers like Google and Facebook, you will need t
...
+3. In the left tab, click **Credentials**.
+4. Click the **Create credentials** then **OAuth client ID**.
```
Comment (ardalis) [inline aspnet/mvc/overview/security/create-an-aspnet-mvc-5-app-with-facebook-and-google-oauth2-and-openid-sign-on.md]: Remove "the".
Or change to Click the **Create credentials** button then...

File: aspnet/mvc/overview/security/create-an-aspnet-mvc-5-app-with-facebook-and-google-oauth2-and-openid-sign-on.md [hunk 3]
```diff
@@ -110,16 +110,16 @@ To connect to authentication providers like Google and Facebook, you will need t
...
-3. In the left tab, click **APIs &amp; auth**, and then &gt; **Credentials**.
-4. Click the **Create New Client ID** under OAuth. 
+1. If you haven't create a project before, select **Credentials** in the left tab, and then select **Create**.
+3. In the left tab, click **Credentials**.
+4. Click the **Create credentials** then **OAuth client ID**. 
 
     1. In the **Create Client ID** dialog, keep the default **Web application** for the application type.
-    2. Set the **Authorized JavaScript** origins to the SSL URL you used above (`https://localhost:44300/` unless you've created other SSL projects)
+    2. Set the **Authorized JavaScript** origins to the SSL URL you used above (`https://localhost:44300` unless you've created other SSL projects)
     3. Set the **Authorized redirect URI** to:
```
Comment (ardalis) [inline aspnet/mvc/overview/security/create-an-aspnet-mvc-5-app-with-facebook-and-google-oauth2-and-openid-sign-on.md]: Should there be consistency between the use of URL (on previous line) and URI (on this line)?

**Comment by Rick-Anderson** [comment]
@ardalis  anything else? Need to merge this so we don't have merge conflicts on our other work.  cc @01binary

**Comment by Rick-Anderson** [comment]
@nkhalili  thanks!

**Pull request merged**